### ISSUE TITLE
to run yolov6_s_syncbn_fast_8xb32-300e_coco.py  but, KeyError:'pred_instances' 

### ISSUE THREAD

**Issue by Youngforever0911** [open-mmlab/mmyolo#424]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmyolo/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmyolo.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmyolo).

### 💬 Describe the reimplementation questions

After modifying your own dataset, you can train, but KeyError:'pred_instances' will appear during verification
This error occurs only when running Yolov6. None of the other networks have this error.
What exactly is the problem here? thanks.

### Environment

    TorchVision: 0.11.2
    OpenCV: 4.7.0
    MMEngine: 0.4.0
    Python: 3.9.15
    PyTorch: 1.10.1
  - CUDA Runtime 11.3

### Expected results

{"lr": 0.0004900000000000001, "data_time": 0.5544450378417969, "loss": 2.906997742652893, "loss_cls": 1.2359778821468352, "loss_bbox": 1.671019868850708, "time": 1.3774438333511352, "epoch": 1, "step": 50}
{"lr": 0.0010799835116769047, "data_time": 0.1432168436050415, "loss": 2.696580023765564, "loss_cls": 1.4010140085220337, "loss_bbox": 1.2955660128593445, "time": 0.8236962747573853, "epoch": 2, "step": 109}
{"lr": 0.001669898018240965, "data_time": 0.15374878406524659, "loss": 2.5058381175994873, "loss_cls": 1.3357281708717346, "loss_bbox": 1.170109944343567, "time": 0.8376589107513428, "epoch": 3, "step": 168}
{"lr": 0.0022596894826848774, "data_time": 0.158615083694458, "loss": 2.3059376287460327, "loss_cls": 1.2314449834823609, "loss_bbox": 1.0744926452636718, "time": 0.842385721206665, "epoch": 4, "step": 227}
{"lr": 0.002849303880035956, "data_time": 0.15253146171569823, "loss": 1.9585838985443116, "loss_cls": 0.8918876135349274, "loss_bbox": 1.0666962921619416, "time": 0.8356842803955078, "epoch": 5, "step": 286}
{"lr": 0.0034386872029107027, "data_time": 0.155863037109375, "loss": 1.7776092672348023, "loss_cls": 0.8634419965744019, "loss_bbox": 0.9141672718524932, "time": 0.8391343688964844, "epoch": 6, "step": 345}
{"lr": 0.004027785467067886, "data_time": 0.1569793701171875, "loss": 1.6874488306045532, "loss_cls": 0.8580239737033843, "loss_bbox": 0.829424865245819, "time": 0.8390152406692505, "epoch": 7, "step": 404}
{"lr": 0.004616544716959636, "data_time": 0.15279102802276612, "loss": 1.5519139194488525, "loss_cls": 0.8287308073043823, "loss_bbox": 0.7231831169128418, "time": 0.8370606088638306, "epoch": 8, "step": 463}
{"lr": 0.005204911031280091, "data_time": 0.1521530866622925, "loss": 1.4532364535331725, "loss_cls": 0.7932368779182434, "loss_bbox": 0.6599995863437652, "time": 0.8353451061248779, "epoch": 9, "step": 522}
{"lr": 0.005792830528511112, "data_time": 0.1595530366897583, "loss": 1.368888783454895, "loss_cls": 0.7678955686092377, "loss_bbox": 0.6009932148456574, "time": 0.8409101724624634, "epoch": 10, "step": 581}

This is the result of training.

### Additional information

The data set has been modified, but it can run through yolov5/7/x, except V6.

**Comment by hhaAndroid** [comment]
@Youngforever0911 Please try the dev branch

**Comment by Youngforever0911** [comment]
> @Youngforever0911请尝试开发分支

Thanks.
